docs-beta/content/quickstart/_index.md

```diff
@@ -0,0 +1,466 @@
+              "cutoffNanos": "0"
+            }
+          ],
+          "shardSetId": 0,
+          "hostname": "localhost",
+          "port": 9000,
+          "metadata": {
+            "debugPort": 0
+          }
+        }
+      },
+      "replicaFactor": 1,
+      "numShards": 64,
+      "isSharded": true,
+      "cutoverTime": "0",
+      "isMirrored": false,
+      "maxShardSetId": 0
+    },
+    "version": 0
+  }
+}
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+Placement initialization can take a minute or two. Once all the shards have the `AVAILABLE` state, the node has finished bootstrapping, and you should see the following messages in the node console output.
+
+<!-- TODO: Fix these timestamps -->
+
+```shell
+{"level":"info","ts":1598367624.0117292,"msg":"bootstrap marking all shards as bootstrapped","namespace":"default","namespace":"default","numShards":64}
+{"level":"info","ts":1598367624.0301404,"msg":"bootstrap index with bootstrapped index segments","namespace":"default","numIndexBlocks":0}
+{"level":"info","ts":1598367624.0301914,"msg":"bootstrap success","numShards":64,"bootstrapDuration":0.049208827}
+{"level":"info","ts":1598367624.03023,"msg":"bootstrapped"}
+```
+
+You can check on the status by calling the _placement_ endpoint:
+
+{{< tabs name="check_placement" >}}
+{{% tab name="Command" %}}
+
+```shell
+curl {{% apiendpoint %}}placement | jq .
+```
+
+{{% /tab %}}
+{{% tab name="Output" %}}
+
+```json
+{
+  "placement": {
+    "instances": {
+      "m3db_local": {
+        "id": "m3db_local",
+        "isolationGroup": "local",
+        "zone": "embedded",
+        "weight": 1,
+        "endpoint": "127.0.0.1:9000",
+        "shards": [
+          {
+            "id": 0,
+            "state": "AVAILABLE",
+            "sourceId": "",
+            "cutoverNanos": "0",
+            "cutoffNanos": "0"
+          },
+          …
+          {
+            "id": 63,
+            "state": "AVAILABLE",
+            "sourceId": "",
+            "cutoverNanos": "0",
+            "cutoffNanos": "0"
+          }
+        ],
+        "shardSetId": 0,
+        "hostname": "localhost",
+        "port": 9000,
+        "metadata": {
+          "debugPort": 0
+        }
+      }
+    },
+    "replicaFactor": 1,
+    "numShards": 64,
+    "isSharded": true,
+    "cutoverTime": "0",
+    "isMirrored": false,
+    "maxShardSetId": 0
+  },
+  "version": 2
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% notice tip %}}
+[Read more about the bootstrapping process](https://docs.m3db.io/operational_guide/bootstrapping_crash_recovery/).
+{{% /notice %}}
+
+## Writing and Querying Metrics
+
+M3DB supports two query engines:
+
+-   **Prometheus (default)** - robust and commonly-used query language for metrics
+-   **M3 Query Engine** - in development higher-performance query engine
+
+### Writing Metrics
+
+As M3DB is a time series database (TSDB), metric data consists of a value, a timestamp, and tags to bring context and meaning to the metric.
```

Comment:
Maybe worth mentioning here that it also supports Graphite/statsd metrics but for the purposes of this quickstart, we'll just focus on Prom formatted metrics?

Reply:
@benraskin92 To clarify here. Does Graphite and statsd use the same format?

Reply:
Yes, so statsd is the metrics format, and graphite is the query language